### ISSUE TITLE
Fix imports to make tests runnable again

### ISSUE THREAD

**Issue by uchchwhash** [GeoscienceAustralia/wagl#356]
The test suite isn't runnable at the moment because of some library upgrades.

`import gdal` does not work anymore, but `from osgeo import gdal` works. Same goes for the `osr` package.

Please look into these files (reported by @sixy6e)

```
tests/test_geobox.py
tests/test_lon_lat.py
wagl/geobox.py
wagl/longitude_latitude_arrays.py
wagl/tiling.py
wagl/unittesting_tools.py
```

**Comment by truth-quark** [comment]
What platform/OS are the tests being run on?

**Comment by sixy6e** [comment]
I attempted to use it with `python-3.9`, `gdal-3.3.0` on `5.12.13-arch1-2`.

**Comment by truth-quark** [comment]
> I attempted to use it with `python-3.9`, `gdal-3.3.0` on `5.12.13-arch1-2`.

The import bug bit with an Ubuntu VM I hacked up for testing. I thought your local archlabs system worked fine.

Did the test runs fail recently on your arch system?

**Comment by uchchwhash** [comment]
The endgame here is to run the tests on GitHub actions whenever there's a commit. So feel free to create an Action because that's basically the "platform" where it needs to work IMHO.

**Comment by uchchwhash** [comment]
Fixed by #358.
